Master Project/Assets/Scenes/Main/IngredientDirectiveScript.cs

```diff
@@ -27,17 +28,16 @@ public class IngredientDirectiveScript : MonoBehaviour {
 		//Debug.Log ("Collision");
 		if (col.gameObject.tag == "ItemOne") {
 			Debug.Log ("first");
+			SceneManager.LoadScene ("Shaking");
 		}
 		else if (col.gameObject.tag == "ItemTwo") {
 			Debug.Log ("second");
+			SceneManager.LoadScene ("Microwave_Goldberg");

```

Comment:
This hard coding is definitely not a long term solution. There is an enum IngredientType (currently only has WhippedCream). What should be done is there should be a Dictionary<IngredientType ingredientType, string SceneName> that maps each ingredient to the proper scene.

Also, each scene is going to need information about what ingredient they're working on.